### PR TITLE
Always return https link when shortening (OPS-4975)

### PR DESCRIPTION
**Pull request by olivierlefloch**

### CODE REVIEW

File: django_short_urls/tests/models_link_test.py [hunk 1]
```diff
@@ -135,3 +136,11 @@ def test_prefix(self):
         prefix = 'foo'
         link, _ = _shorten(self.URL, prefix=prefix)
         self.assertEqual(link.prefix, prefix)
+
+    def test_build_absolute_uri(self):
+        link, _ = _shorten("http://www.work4labs.com/", short_path='foo')
+
+        self.assertEqual(
+            link.build_absolute_uri(RequestFactory().get('/test')),
+            "https://testserver/foo"
```
Comment (asiebert) [inline django_short_urls/tests/models_link_test.py]: I'm so puzzled about it being the first test actually checking the full URI − means there is no regression test here.

File: django_short_urls/models.py [hunk 2]
```diff
@@ -161,8 +161,10 @@ def build_relative_path(self):
         return '/%s' % self.hash
 
     def build_absolute_uri(self, request):
-        """Builds the absolute url for the target link (including full server url)"""
-        return request.build_absolute_uri(self.build_relative_path())
+        """Builds the absolute url HTTPs for the target link (including full server url)"""
```
Comment (asiebert) [inline django_short_urls/models.py]: It's already better than "use the scheme of the incoming API request", though ideally we could have an option for "force HTTPS", or "use the same scheme as the long_url shortened"?

Comment (olivierlefloch) [inline django_short_urls/models.py]: No need right now

File: django_short_urls/tests/models_link_test.py (hunk 1 above)
Comment (olivierlefloch) [inline django_short_urls/tests/models_link_test.py]: :white_check_mark:

**Pull request merged**